yb-voyager/cmd/common.go

```diff
@@ -248,25 +248,19 @@ func displayExportedRowCountSnapshot(snapshotViaDebezium bool) {
 		}
 		keys := lo.Keys(exportedRowCount)
 		sort.Strings(keys)
-		leafPartitions := getLeafPartitionsFromRootTable(keys)
 		for _, key := range keys {
-			if source.Schema != "" {
-				tableParts := strings.Split(key, ".")
-				table := tableParts[0]
-				schema, _ := getDefaultSourceSchemaName() // err can be ignored as these table names will be qualified for non-public schema
-				if len(tableParts) > 1 {
-					schema = tableParts[0]
-					table = tableParts[1]
-				}
-				displayTableName := table
-				if source.DBType == POSTGRESQL && leafPartitions[key] != nil {
-					partitions := strings.Join(leafPartitions[key], ", ")
-					displayTableName = fmt.Sprintf("%s (%s)", table, partitions)
-				}
-				uitable.AddRow(schema, displayTableName, exportedRowCount[key])
-			} else {
-				uitable.AddRow(source.DBName, key, exportedRowCount[key])
+			table, err := namereg.NameReg.LookupTableName(key)
+			if err != nil {
+				utils.ErrExit("lookup table %s in name registry : %v", key, err)
+			}
+			displayTableName := table.CurrentName.Unqualified.MinQuoted
```

Comment:
I had imagined that at very few places we would actually referring to the specific kinds of names (e.g. `CurrentName.Unqualified.MinQuoted`). Instead, `NameTuple` should have purpose driven name methods e.g. `ForKey()`, `ForUserQuery()`, `ForLogging()`, `ForDisplay()`, `AsOra2pgOutputFile()`. That way, all the name related decisions are captured at one place.